Made some changes so this runs on 4.12+ cluster

afouladi7 opened the pull request:
Adding all these changes to ensure this demo still runs on 4.12+ cluster. Also tested on 4.13.

afouladi7 committed:
adding all the changes need for this to work in a 4.12+ cluster

afouladi7 committed:
Delete users.htpasswd

afouladi7 committed:
added the changes for gitops and bc for cosign

afouladi7 committed:
Merge branch 'main' of https://github.com/afouladi7/devsecops-demo

afouladi7 committed:
adding all the changes for 4.12+

afouladi7 committed:
changing the gitops app config

rcarrata commented on bootstrap/roles/ocp4-install-gitops/tasks/gitops.yaml:
```diff
@@ -53,35 +53,39 @@
-    until: crds.resources|length > 0
-    retries: 30
-    delay: 10
+# - name: Adapt to the openshift_cluster_version LESS than 4.9
```
@afouladi7 why we are removing these checks? I've tested in 4.13 and worked like a charm.

afouladi7 replied:
Are you running this on BM? I tested this on both BM and RHPDS on 4.12.

rcarrata replied:
VMs (Hetzner + libvirt + 4.13) and I'll do it in RHPDS 4.11 as well :)

rcarrata commented on bootstrap/roles/ocp4-install-pipelines/templates/task-s2i-java-11.yaml.j2:
```diff
@@ -109,6 +109,10 @@ spec:
+      securityContext:
+        capabilities:
+          add:
+            - SETFCAP
```
@afouladi7 tested in my environment and worked like a charm. Thanks!

rcarrata commented on bootstrap/roles/ocp4-post-acs/tasks/post_ci.yaml:
```diff
@@ -47,6 +47,39 @@
         namespace: "{{ ocp4_demo_pipeline_namespace }}"
       type: Opaque
 
+# - name: Get Service Account objects
```
I've fixed (check the main branch) grabbing directly the secret token itself, instead of grepping for the sa pipeline token. Shall we move to the main branch proposal and avoid to use the shell commands?

rcarrata reviewed (changes requested):
Kindly rollback to use the OpenShift Pipeline checks again, adapt to the post_ci to use the main branch approach. 

Thanks for your contributions @afouladi7!

afouladi7 committed:
adiing sbom to the pipeline

rseip-rh committed:
Updated ACS policy enforcement guidance to permit initial pipeline to
run without failure.

cmays20 committed:
Create trust_quay_from_another_cluster.md

rseip-rh committed:
Adds single command for Fedora to install all prerequisite tools.

afouladi7 committed:
adding sbom signing and supporting tasks

afouladi7 committed:
Update cosign-build.yaml.j2

changing the ubi to 8.8

afouladi7 committed:
changing to the latest ubi

afouladi7 committed:
changing to the latest ubi

afouladi7 committed:
adding sbom, attestion, image verify to demo.sh

afouladi7 committed:
adding signing to demo.sh

afouladi7 committed:
adding signing to demo.sh

afouladi7 committed:
Merge pull request #1 from redhat-dod-sa/rseip-doc-edits

doc edits for ACS policy change and Fedora prerequisite installs

afouladi7 committed:
updating gogs and quay

kincl committed:
Fixing ArgoCD CR: Cannot specify both keycloak and dex

We cannot specify both keycloak and dex as SSO providers. When I tried
enabling just keycloak the gitops-operator hit some segfault errors so
switch to just using dex for now

afouladi7 committed:
Merge pull request #2 from kincl/fix-argocd

Fixing ArgoCD CR: Cannot specify both keycloak and dex

kincl committed:
Removing update to add keycloak for now

Using Keycloak breaks the gitops operator as of v1.9.1

afouladi7 committed:
fixing the argo issue with OCP dex

afouladi7 committed:
Merge branch 'afouladi7:main' into main

afouladi7 committed:
updated the readme for the credits

afouladi7 committed:
Merge branch 'afouladi7:main' into main

kincl committed:
Adding Quay to doc and status.sh

afouladi7 committed:
Merge pull request #3 from kincl/fix-argocd

Removing update to add keycloak for now

afouladi7 committed:
Merge pull request #4 from kincl/add-quay-doc

Adding Quay to doc and status.sh

afouladi7 committed:
Merge pull request #2 from redhat-dod-sa/main

adding the argo dex fix

afouladi7 committed:
adding secure route for gogs

cmays20 committed:
Tell it to ignore cert validation for the gogs url

afouladi7 committed:
Merge pull request #6 from cmays20/patch-2

Tell it to ignore cert validation for the gogs url

afouladi7 committed:
Merge pull request #1 from cmays20/patch-1

Create trust_quay_from_another_cluster.md

afouladi7 committed:
putting quay and acs on a diet

afouladi7 committed:
Merge pull request #3 from redhat-dod-sa/main

adding charges from org

afouladi7 committed:
Update cicd-gogs.yaml.j2

afouladi7 committed:
Update cicd.yaml

afouladi7 committed:
adding console link for acs

afouladi7 committed:
adding the https for gogs

afouladi7 committed:
adding the https for gogs

afouladi7 committed:
adding the https for gogs

afouladi7 committed:
adding the https for gogs

afouladi7 committed:
adding the https for gogs

afouladi7 committed:
Update quay-subscription.yaml.j2
